docs/components/connectors/out-of-the-box-connectors/google-maps-platform.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: google-maps-platform
-title: Google Maps Platform
-sidebar_label: GitHub Webhook
+title: Google Maps Platform Connector
+sidebar_label: Google Maps Platform Connector
```

Comment:
This was showing correctly named in the Optimize section of the site, but incorrectly named in the main section of the site 😅